2200028/4.py

```diff
@@ -0,0 +1,15 @@
+import sys
+sys.stdin = open("input.txt", "r")
+T = int(input())
+
+for t in range(1, T+1):
+    N = 2
+    for _ in range(2):
+        avg = 0
+        num_list= list(map(int,input().split()))
+        result = []
+        avg = sum(num_list)/len(num_list)
+        for i in num_list:
+            if i <= avg:
+                result.append(i)
```

Comment:
잘 풀어주셨네요! 👍 한 가지 수정해야 할 점을 말씀드릴게요.
```suggestion
    N = int(input())
    num_list= list(map(int,input().split()))
    result = []
    avg = sum(num_list)/len(num_list)
    for i in num_list:
        if i <= avg:
            result.append(i)
```
`num_list`라는 입력 값은 한 번만 주어지므로 `for` 문으로 두 번 받는 구조는 수정할 필요가 있습니다. `for`문을 제거한 뒤 `N`과 `num_list`를 차례대로 받도록 수정해주시면 좋을 것 같습니다! 😄